style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.background {
+  background-image: url("images/header-shapes-mobile.png");
+  background-position: 0% 100%;
+}
+
+.navbar {
+  background-color: white;
+  color: #6070ff;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 18px;
+  height: 80px;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  width: 150px;
+  height: 20px;
+  font-weight: 700;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.bold {
+  color: #172b4d;
+}
+
+p {
+  color: #344563;
+}
+
+.next-color {
+  color: #7f8cff;
+  font-size: 16px;
+  font-weight: 500;
+  margin-top: 10px;
+}
+
+.navbar a {
+  text-decoration: none;
+}
+
+.background-image {
+  padding-top: 70px;
+}
+
+.onscreen {
+  text-align: left;
+  position: absolute;
+  font-size: 16px;
+  margin: 90px 50px 20px 50px;
```

Comment:
Good job so far by adding some of the changes required by the previous reviewers. So, I noticed that the previous reviewers requested to try to check on your design and figma design, there is a bit different, and you are supposed to have something closer or similar. 
Please, for each element : add the correct color, the correct font-family and the correct margin or padding

Your design
<img width="444" alt="image" src="https://user-images.githubusercontent.com/7243132/170592896-e62c0f59-1ecc-4091-8c42-9a10e0df1eee.png">

Figma:

<img width="282" alt="image" src="https://user-images.githubusercontent.com/7243132/170593035-afce6f7c-fe45-453a-ac8c-cab0a812e4ff.png">